README.md

```diff
@@ -128,6 +128,70 @@ Note that `Nil` is a type that has one value `nil`.
 - Map entry is represented as a two-element vector.
 - resolving unbound var returns `nil`, not the value `Unbound`. You can still check if the var is bound with `bound?` function.
 
+### Floating-point Constants and the BigFloat Type
```

Comment:
I really appreciate this thorough explanation. I do think this is a fairly niche topic that is probably better placed in a separate file in `docs/misc` directory and linked from here.

Reply:
Done!